### ISSUE TITLE
Cannot run on Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by williamkapke** [nodejs/node-chakracore#156]
I need to install using `n`... which is possible using:
```
PROJECT_NAME="node" PROJECT_URL="https://nodejs.org/download/chakracore-nightly/" n project v7.0.0-nightly201612169538b4aa4f
```

... which almost works. It downloads it but ends with:

```
node: error while loading shared libraries: libunwind.so.8: cannot open shared object file: No such file or directory
```

The same error happens if you then try to run `node`. So, it seems it installs it- but can't run it.

Although i can't use `nvs`, I tried the directions in the README anyhow... but got:
```
~ $ . "$NVS_HOME/nvs.sh" install
Downloading bootstrap node from https://nodejs.org/dist/v6.9.1/node-v6.9.1-linux-x64.tar.xz
################################################################################################################################### 100.0%
mv: cannot stat '/app/.nvs/cache/node-v6.9.1-linux-x64/bin/node': No such file or directory
rm: cannot remove '/app/.nvs/cache/node-v6.9.1-linux-x64': No such file or directory
Failed to download boostrap node binary.
```

Building from sources is not an option.

**Comment by kunalspathak** [comment]
//cc : @jasongin

**Comment by jianchun** [comment]
Latest chakracore master has removed libunwind dependency. Sync to chakracore master would resolve this problem.

**Comment by jasongin** [comment]
To resolve the `libunwind.so` dependency with the current node-chakracore releases you can run this command:
```sh
sudo apt-get install libunwind-dev
```

@williamkapke I'm unable to reproduce that nvs error on Ubuntu 16.04. It looks like you have set `$NVS_HOME` to `/app/.nvs`, is that right? Does the `/app/.nvs/cache` directory exist? Was that `.xz` file fully downloaded? The size should be 9351072 bytes.

Regardless, even if nvs works it is not going to affect the `libunwind` dependency of node-chakracore.

**Comment by williamkapke** [comment]
@jasongin:
![image](https://cloud.githubusercontent.com/assets/739813/21276069/9bac61b2-c384-11e6-88cf-398c41947980.png)

...and:
![image](https://cloud.githubusercontent.com/assets/739813/21276122/e0c05cae-c384-11e6-86f7-14e43f838b22.png)
(sudo doesn't exist on Heroku)

**Comment by jasongin** [comment]
@williamkapke Try running `apt-get update` to refresh your package index first, afterward it should find the `libunwind-dev` package.

The NVS install script is trying to run the these commands:
```sh
cd /app/.nvs/cache
tar -Jxvf node-v6.9.1-linux-x64.tar.xz node-v6.9.1-linux-x64/bin/node
mv node-v6.9.1-linux-x64/bin/node .
rm -r node-v6.9.1-linux-x64
```
From the error output, it looks like the `tar` command did not extract the `node` binary as expected. Can you share the output from `tar --version`? I have: `tar (GNU tar) 1.28`

**Comment by williamkapke** [comment]
Oh yes- now I remember why I was only able to get `n` working on Heroku: because it is a locked down file system except for the `/app` folder (which is also $HOME).

![image](https://cloud.githubusercontent.com/assets/739813/21280257/abfb12e2-c399-11e6-8cfd-b8d96e2bc3c0.png)

Anyhow- so, I can't use `nvs`... that's a different issue I think.

**Comment by williamkapke** [comment]
Any update on this?

**Comment by digitalinfinity** [comment]
@williamkapke thanks for following up. This was fixed in Microsoft/ChakraCore#2086, and will be fixed in Node-ChakraCore when we take a new drop of ChakraCore (likely in the next couple weeks once ChakraCore stabilization progresses).

**Comment by williamkapke** [comment]
```
~ node
> process.versions
{ http_parser: '2.7.0',
  node: '8.0.0-nightly2017022305e0f044bc',
  chakracore: '2.0.0.0',
  uv: '1.10.1',
  zlib: '1.2.8',
  ares: '1.10.1-DEV',
  modules: '51',
  openssl: '1.0.2j' }
>
```

... ooohh DANG! I'm in business :tada:


THANKS!!!!!!
